lib/WebSocket.js

```diff
@@ -267,28 +263,23 @@ class WebSocket extends EventEmitter {
       return;
     }
 
-    try {
-      this.readyState = WebSocket.CLOSING;
-      this._closeCode = code;
-      this._closeMessage = data;
-      var mask = !this._isServer;
-      this._sender.close(code, data, mask, (err) => {
```

Comment:
This can throw when `WebSocket#close()` is called with an invalid status code, but I think it's better to not wrap this in a `try...catch` as it is a programmer error.

`WebSocket#close()` is never called internally with an invalid status code.